cmd/client.go

```diff
@@ -92,14 +95,29 @@ func (cli *Client) GetJobSpecs(c *clipkg.Context) error {
 func (cli *Client) CreateJobSpec(c *clipkg.Context) error {
 	cfg := cli.Config
 	if !c.Args().Present() {
-		return cli.errorOut(errors.New("Must pass in job JSON"))
+		return cli.errorOut(errors.New("Must pass in JSON or filepath"))
+	}
+	arg := c.Args().First()
+	var buf *bytes.Buffer
+	if gjson.Valid(arg) {
+		buf = bytes.NewBufferString(arg)
+	} else {
+		dir, err := homedir.Expand(arg)
+		if err != nil {
+			return cli.errorOut(err)
+		}
+		file, err := ioutil.ReadFile(dir)
+		if err != nil {
+			return cli.errorOut(err)
+		}
+		buf = bytes.NewBuffer(file)
```

Comment:
Let's place this entire `else` into a a `fromFile` method to improve readability.